src/test/java/org/apache/mesos/scheduler/recovery/RecoverySchedulerTest.java

```diff
@@ -41,7 +41,7 @@
  * as failed
  */
 public class RecoverySchedulerTest {
-    private DefaultRecoveryScheduler recoveryScheduler;
+    private DefaultRecoveryScheduler repairScheduler;
```

Comment:
Also, I'm confused.  I thought [I updated these already](https://github.com/mesosphere/dcos-commons/blob/master/src/test/java/org/apache/mesos/scheduler/recovery/RecoverySchedulerTest.java#L44).  @mgummelt: What's going on with the diff here?  Regardless, let's leave changes irrelevant to the content of this PR to other PRs.